web-www/src/sections/nct/TokenSection.tsx

```diff
@@ -0,0 +1,215 @@
+import React from 'react';
+import { graphql, useStaticQuery } from 'gatsby';
+import { Box, Typography } from '@mui/material';
+import SanityImage from 'gatsby-plugin-sanity-image';
+
+import Title from 'web-components/lib/components/title';
+import {
+  BlockContent,
+  SanityBlockContent,
+} from 'web-components/src/components/block-content';
+import Section from 'web-components/lib/components/section';
+import { formatDate } from 'web-components/lib/utils/format';
+import ContainedButton from 'web-components/lib/components/buttons/ContainedButton';
+import OutlinedButton from 'web-components/lib/components/buttons/OutlinedButton';
+import { Label } from 'web-components/lib/components/label';
+import Countdown from 'web-components/lib/components/countdown';
+
+import type { NctTokenSectionQuery } from '../../generated/graphql';
+
+const query = graphql`
+  query nctTokenSection {
+    sanityNctPage {
+      launchDate
+      tokenSection {
+        cardImage {
+          ...customImageFields
+        }
+        cardTitle
+        cardSubtitle
+        _rawCardBody
+        cardButton {
+          ...buttonFields
+        }
+        detailImage {
+          ...customImageFields
+        }
+        detailTitle
+        detailSubtitle
+        _rawDetailBody
+        detailButton1 {
+          ...buttonFields
+        }
+        detailButton2 {
+          ...buttonFields
+        }
+      }
+    }
+  }
+`;
+
+const BodyContent: React.FC<{ content: SanityBlockContent }> = ({
+  content,
+}) => (
+  <BlockContent
+    noYMargin
+    content={content}
+    sxWrap={{ '& p': { fontSize: [18, 22] } }}
+  />
+);
+
+export const TokenSection = (): JSX.Element => {
+  const { sanityNctPage } = useStaticQuery<NctTokenSectionQuery>(query);
+  const data = sanityNctPage?.tokenSection;
+  const launchDate = sanityNctPage?.launchDate;
+
+  const btnPad = { px: { xs: 8, sm: 12 } } as const;
```

Comment:
note: if you want to extract `sx` styles into a variable, you need to declare it `as const` to satisfy typescript

Reply:
nice tip